diff --git a/lib/ImageHelper.php b/lib/ImageHelper.php
index 1b36eb8..cf84e4b 100644
--- a/lib/ImageHelper.php
+++ b/lib/ImageHelper.php
@@ -369,7 +369,6 @@ class ImageHelper {
			// therefore using site_url() would lead to a wrong content/ path
			if ( false === strpos($upload_dir['baseurl'], site_url()) ) {
				// use HOME_URL and relative image path
-				$tmp = get_home_url().$tmp;
			} else if ( !$result['absolute'] ) {
				$tmp = site_url().$tmp;
			}
